Build.bee.cs

```diff
@@ -58,22 +58,29 @@ public TundraNativeProgram(string name) : base(name)
 
     static NPath GenerateGitFile()
     {
-        var result = Shell.Execute("git for-each-ref --count 1 --format \"%(objectname):%(refname:short)\"");
-        if (!result.Success)
-            return null;
-
-        var matches = Regex.Matches(result.StdOut, @"(\w+?):(.+)");
-        if (matches.Count == 0)
+        try
```

Comment:
Hmm, probably all of this can be replaced with a call to `Bee.BuildTools.VersionControl.SetupWriteGitRevisionInfoFile` ?

Reply:
These don't seem to be doing the same thing, e.g. here it puts that into `generated/git_rev.c`